From b0c63502f004db68b485354967bb1c56c071f4eb Mon Sep 17 00:00:00 2001
From: Adrian Perez de Castro <aperez@igalia.com>
Date: Tue, 31 May 2022 00:48:21 +0300
Subject: [PATCH] Build failure when cross-building for 64-bit ARM
 https://bugs.webkit.org/show_bug.cgi?id=241109

Unreviewed build fix.

* Source/WebCore/bindings/js/JSDOMMapLike.cpp: Add missing
  JavaScriptCore/HashMapImplInlines.h header inclusion.

Signed-off-by: Adrian Perez de Castro <aperez@igalia.com>
Upstream status: https://github.com/WebKit/WebKit/pull/1165
---
 Source/WebCore/bindings/js/JSDOMMapLike.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WebCore/bindings/js/JSDOMMapLike.cpp b/Source/WebCore/bindings/js/JSDOMMapLike.cpp
index e132c39fa54..2cb4b1b59a3 100644
--- a/Source/WebCore/bindings/js/JSDOMMapLike.cpp
+++ b/Source/WebCore/bindings/js/JSDOMMapLike.cpp
@@ -28,6 +28,7 @@
 
 #include "WebCoreJSClientData.h"
 #include <JavaScriptCore/CatchScope.h>
+#include <JavaScriptCore/HashMapImplInlines.h>
 #include <JavaScriptCore/JSMap.h>
 #include <JavaScriptCore/VMTrapsInlines.h>
 
-- 
2.36.1

